src/extension.ts

```diff
@@ -203,8 +209,36 @@ export async function activate(context: vscode.ExtensionContext) {
     kicsScanCommand.registerKicsRemediation();
     // Refresh sca tree with start scan message
     scaResultsProvider.refreshData(constants.scaStartScan);
+    
+    //You will need to add a checkbox to enable the extension's activation
+    const isPromptEnabled:boolean = false;
+    //You will need to change it to recieve a port from the user
+    const port:number = 3312;
```

Comment:
please extract it to a private method ("registerPromptListener" ?)

Reply:
Added it, and committed it now, let me know if it wasn't what you meant, or you need it to have some different functionality.

Reply:
What I meant is about ine 212 to 235, all the logic. that we will have there only one line for registration

Reply:
Hi @OrShamirCM and @tiagobcx , can you please review the changes I made on Sunday?